### PR TITLE
Add useGlobal documentation

### PR DESCRIPTION
**Pull request by angelod1as**

### CODE REVIEW

File: docs/docs/use-global.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+const initialState = {
+  menuOpen: false, // example
+  searchTerm: undefined,
+}
+
+const actions = {
+  toggleMenu: (store) => { // store is always passed as first parameter
+    store.setState({ menuOpen: !store.state.menuOpen })
+  },
+  doSearch: (store, searchTerm) => { // second parameter is received by useGlobal hook
```
Comment (gustavoguichard) [inline docs/docs/use-global.md:33]:
```suggestion
  doSearch: (store, searchTerm) => { // the second and following parameters are the ones you'll send when using this action
```

Comment (gustavoguichard) [reply docs/docs/use-global.md:33]: Not sure it is the best way to put it, but I think it was a little abstract... any ideas on how to improve this line?

**Review by gustavoguichard — APPROVED** [review]
L❤️TM

File: docs/docs/use-global.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+const actions = {
+  toggleMenu: (store) => { 
+    // store is always passed as first parameter
+    store.setState({ menuOpen: !store.state.menuOpen })
+  },
+  doSearch: (store, searchTerm) => { 
+    // you can send the second (and following) parameter on the action function, eg:
+    // doSearch(searchTerm)
```
Comment (angelod1as) [inline docs/docs/use-global.md:36]: @gustavoguichard what do you think?

File: docs/docs/use-global.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+---
+id: use-global
+title: The useGlobal hook
+---
+
+Croods has a global state manager that can be of use if there's need for manipulating globally available state. 
+
+With it you can save and request information from this store without needing to tamper with Croods store manually, **which is highly disencouraged**.
```
Comment (giovanisleite) [inline docs/docs/use-global.md:8]:
```suggestion
With it you can save and request information from this store without needing to tamper with Croods store manually, **which is highly discouraged**.
```

**Review by giovanisleite — APPROVED** [review]

**Pull request merged**